.github/workflows/release.yml

```
@@ -38,17 +38,22 @@ jobs:
         with:
           tag-template: "{VERSION}"
           version-command: |
-            cat current-version.txt
+            cat galaxy.yml | grep version: | cut -d' ' -f2
 
       - name: Install Ansible
         run: |
           pip install --constraint=.github/workflows/constraints.txt ansible
 
-      - name: Trigger a new import on Galaxy.
+      - name: Release to Ansible Galaxy.
+        if: "steps.check-version.outputs.tag"
+        env:
+          ANSIBLE_GALAXY_TOKEN: ${{ secrets.GALAXY_TOKEN }}
         run: >-
-          ansible-galaxy role import
-          --api-key ${{ secrets.GALAXY_TOKEN }} $(echo ${{ github.repository }}
-          | cut -d/ -f1) $(echo ${{ github.repository }} | cut -d/ -f2)
+          ansible-playbook -i 'localhost,' galaxy-deploy.yml
+          -e "github_tag=$(cat galaxy.yml|grep version:|cut -d' ' -f2)"
```

I missed it here, same as above.
```suggestion
          -e "github_tag=$(cat galaxy.yml | grep version: | cut -d' ' -f2)"
```

---

Done.